build.gradle

```diff
@@ -16,6 +16,7 @@ ext {
 
     appCompatVersion = '1.0.2'
     coreVersion = '1.3.2'
+    fragmentKtx = '1.4.1'
```

Comment:
`fragmentKtxVersion` may be in line with the others

Reply:
bb978582a5643d17ff332c2ddec084515f3acd35
I changed `fragment-ktx` version to be same with `fragment` version.